hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ConfigCheck.java

```diff
@@ -208,7 +202,7 @@ public void writeData(ObjectDataOutput out) throws IOException {
     @Override
     public void readData(ObjectDataInput in) throws IOException {
         groupName = in.readUTF();
-        groupPassword = in.readUTF();
+        in.readUTF();
```

Comment:
Yes the plan is to stop broadcasting in 3.9. So for 3.8.2 we remove validation, to keep it backwards compatible and rolling upgrades compatible.